dynamallow/model.py

```diff
@@ -193,22 +194,15 @@ def put_batch(cls, *items, **batch_kwargs):
                 {"hash_key": "three"},
             )
         """
-        data, errors = cls.Schema.load(items, many=True)
-        if errors:
-            raise ValidationError("Failed to put batch items", errors=errors)
-        return cls.Table.put_batch(*data, **batch_kwargs)
+        return cls.Table.put_batch(*cls.validate(*items), **batch_kwargs)
 
     @classmethod
     def new_from_raw(cls, raw):
         """Return a new instance of this model from a raw (dict) of data that is loaded by our Schema
 
         :param dict raw: The attributes to use when creating the instance
         """
-        data, errors = cls.Schema.load(raw)
```

Comment:
See my comments above, about raw.  I wanted to add some more color here...

The reason things are structured this way is so that we can validate BEFORE the object is created.  This way we end up with a `raw` object that has EVERYTHING and `**data` is ONLY what validated from the Schema, and thus should be an attribute on the final object.